app/components/domains-table.tsx

```diff
@@ -141,7 +161,7 @@ export default function DomainsTable(props: DomainsTableProps) {
                         <Td>{renderDomainStatus(domain.status)}</Td>
                         <Td>
                           <Flex justifyContent="space-between" alignItems="center">
-                            {domain.name}
+                            {renderDnsRecordName(domain.name)}
```

Comment:
I'm confused why this isn't a component?

Reply:
I don't have perfect answer for this, I just used to use this pattern of table components with render functions which I have seen in many other places.

Reply:
I don't think this is a good pattern.  React provides functional components for exactly this use case.  Why step outside of its component system?

Where I'd use a function call is to some non-React code that isn't returning a component.  But that's exactly what you're doing here, so it doesn't make sense to avoid React.